### ISSUE TITLE
"Remove" option is not shown when invoking from the console

### ISSUE THREAD

**Issue by allentiak** [johannesjo/linux-window-session-manager#57]
Only save, restore and list are shown.

## Your Environment
* Version used: 3.0.8

## Expected Behavior
As a User, when invoking `lwsm` or `lwsm --help`, I should get the documentation for all commands: `list`, `save`, `restore` and `remove`.

## Current Behavior
Only the three first ones (`list`, `save` and `restore`) are shown. `remove` is not.

## Steps to Reproduce (for bugs)
On a working installation, just run `lwsm` or `lwsm --help`

**Comment by johannesjo** [comment]
You're absolutely right. Thanks for reporting! Would you mind providing a PR for this?
